src/ui/profile_preferences.ui

```diff
@@ -576,19 +576,6 @@
              <string>User dictionary:</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_userDictionary">
-             <item>
-              <widget class="QRadioButton" name="radioButton_userDictionary_none">
-               <property name="toolTip">
-                <string>&lt;p&gt;A user dictionary will not be used on the main console command line. Lua commands that target a user dictionary will return &lt;tt&gt;nil&lt;/t&gt; and &lt;i&gt;an error message&lt;/i&gt;.&lt;/p&gt;&lt;p&gt;It will still, however, be possible to access the main dictionary selected above.&lt;/p&gt;</string>
```

Comment:
:bulb: Actually, you could have achieved your objective just by turning ***off*** the *translations* of this and the next (the `text` property) content - which will just change the opening `<string>` tags of the elements to `<string notr="true">` ...!

Reply:
I feel we should remove none now, nobody has asked for it and it just takes up real estate.

Good to know how to turn off translations in .ui files though. Could you add that knowledge to wiki? @SlySven

Reply:
Just clear the `translatable` checkbox on **non-empty** text fields (the Qt Designer plug in will not tend to bother to include fields that are unchanged and thus dim {i.e. not **bold**} - BTW the left arrow on the extreme right of some settings will reset the item to the *default* state) - see all those ones in the properties part of the details for the selected item in here for example:

![Screenshot_20190405_003632](https://user-images.githubusercontent.com/6163092/55595240-e4cef980-573a-11e9-8ec2-a4af494ddff8.png)